src/test/regress/expected/partition_locking_optimizer.out

```diff
@@ -284,11 +284,14 @@ insert into partlockt values(1), (2), (3);
 insert into partlockt values(1), (2), (3);
 insert into partlockt values(1), (2), (3);
 select * from locktest_master where coalesce not like 'gp_%' and coalesce not like 'pg_%';
- coalesce  |       mode       | locktype |  node  
------------+------------------+----------+--------
- partlockt | AccessShareLock  | relation | master
- partlockt | RowExclusiveLock | relation | master
-(2 rows)
+     coalesce      |       mode       | locktype |  node  
```

Comment:
Should we add 
```
not(coalesce like 'partlockt' and mode like 'AccessShareLock');
```

Reply:
origin commit does not, I suggest not add.